libraries/androidutils/src/main/kotlin/io/element/android/libraries/androidutils/bitmap/Bitmap.kt

```diff
@@ -64,7 +64,7 @@ fun BitmapFactory.Options.calculateInSampleSize(desiredWidth: Int, desiredHeight
  * Decodes the [inputStream] into a [Bitmap] and applies the needed rotation based on [orientation].
  * This orientation value must be one of `ExifInterface.ORIENTATION_*` constants.
  */
-fun Bitmap.rotateToMetadataOrientation(orientation: Int): Bitmap {
+fun Bitmap.rotateToExifMetadataOrientation(orientation: Int): Bitmap {
```

Comment:
While I was experimenting with this, I realised this method takes an `Int` value here that could be mistaken for rotation degrees coming from `MediaMetadataRetriever`, so I renamed it to mention the `ExifMetadata` values which should be used instead.